### PR TITLE
fix: enable user retirement sink

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR adds the retirement sink to the default PII models and models

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: tutoraspects/plugin.py [hunk 1]
```diff
@@ -46,9 +46,12 @@
...
         (
             "EVENT_SINK_CLICKHOUSE_MODELS",
-            ["course_overviews", "user_profile", "external_id"],
+            ["course_overviews", "user_profile", "external_id", "auth_user"],
+        ),
+        (
+            "EVENT_SINK_CLICKHOUSE_PII_MODELS",
+            ["user_profile", "external_id", "auth_user"],
         ),
-        ("EVENT_SINK_CLICKHOUSE_PII_MODELS", ["user_profile", "external_id"]),
```
Comment (pomegranited) [inline tutoraspects/plugin.py:51]: Ahh... I'm sorry I missed this change!

@Ian2012 Do you mind to change the names of the variables to make them a little less confusing?  Would prefer:

* `EVENT_SINK_MODELS_ENABLED`
* `EVENT_SINK_PII_MODELS`

Comment (Ian2012) [reply tutoraspects/plugin.py:51]: Sure

**Review by bmtcril — APPROVED** [review]

**Review by pomegranited — APPROVED** [review]
:+1: Works for me too :)

- [x] I tested this on my Tutor dev stack
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A
- [x] User-facing strings are extracted for translation

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-aspects&entry.867055334=https://github.com/openedx/tutor-contrib-aspects/pull/544&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-12-15+20:31&entry.1917517419=2024-01-11+14:48&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**